### PR TITLE
PAYARA-2821 health check checks send multiple notifications when a new notifier is added (Payara 4)

### PR DESCRIPTION
**Pull request by mulderbaba**
Signed-off-by: Mert Caliskan <mert.caliskan@payara.fish>

### CODE REVIEW

**Comment by mulderbaba** [comment]
jenkins quick test payara 4 JDK7 please

File: nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/HealthCheckRebooter.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    @Override
+    public void execute(AdminCommandContext context) {
+        final ActionReport actionReport = context.getActionReport();
+        Properties extraProperties = actionReport.getExtraProperties();
+        if (extraProperties == null) {
+            extraProperties = new Properties();
+            actionReport.setExtraProperties(extraProperties);
+        }
+
+        if (server.isDas()) {
```
Comment (MarkWareham) [inline nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/HealthCheckRebooter.java]: Could perhaps reduce complexity with:
```
if(server.isNotDas() || targetUtil.getConfig(target).isDas()){
	service.reboot();
}
```

**Comment by Pandrex247** [comment]
Jenkins test please

**Comment by Pandrex247** [comment]
Jenkins quick test Payara 4 JDK7 please

File: nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/HealthCheckRebooter.java (hunk 1 above)
Comment (mulderbaba) [reply nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/HealthCheckRebooter.java]: make sense, will apply it in a separate PR by going through all asadmin comands.

**Pull request merged**